dir.props

```diff
@@ -114,7 +114,7 @@
   <!-- Dependency auto-upgrade configuration -->
   <ItemGroup>
     <DependencyBuildInfo Include="CoreFx">
-      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/$(DependencyBranch)</RawUrl>
+      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/dev/api</RawUrl>
```

Comment:
The best I can think of is passing in a `CurrentBranch` property from the subscription, and using that here.

With full-version dependency verification, since it tries to use this url, we need a way for devs to build without having to pass in this property--and we can't check in the branch name because once we do that, we lose the possibility of a "nice" merge. Maybe if the "current" info, exclusively auto-updated, looked like this, it could work:
```
<PropertyGroup>
  <CoreFxCurrentRef>32e4abc8afc7a18484b4bc1de351336b07cdb460</CoreFxCurrentRef>
  <CoreFxCurrentBranch>dev/api</CoreFxCurrentBranch>
  <CoreClrCurrentRef>ff0cf2de609c136085376363beb5ad7eaac02168</CoreClrCurrentRef>
  <CoreClrCurrentBranch>master</CoreClrCurrentBranch>
  <ExternalCurrentRef>e39a63f68adeb45d0ef1a7a157cbf13c89ce3d2d</ExternalCurrentRef>
  <ExternalCurrentBranch>master</ExternalCurrentBranch>
</PropertyGroup>
```

Merges still aren't totally clean because they would have dev/api package dependencies until the next auto-PR cycle, but it's as clean as I can think of.